pkg/code-generator/codegen/templates/kube/verify_codegen.go

```diff
@@ -0,0 +1,47 @@
+package kube
+
+import (
+	"text/template"
+
+	"github.com/solo-io/solo-kit/pkg/code-generator/codegen/templates"
+)
+
+var VerifyGenerateTemplate = template.Must(template.New("kube_verify").Funcs(templates.Funcs).Parse(`
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+CI_ROOT=$(dirname "${BASH_SOURCE[0]}")
+
+DIFFROOT="${SCRIPT_ROOT}/pkg"
+TMP_DIFFROOT="${SCRIPT_ROOT}/_tmp/pkg"
+_tmp="${SCRIPT_ROOT}/_tmp"
+
+cleanup() {
+  rm -rf "${_tmp}"
+}
+trap "cleanup" EXIT SIGINT
+
+cleanup
+
+mkdir -p "${TMP_DIFFROOT}"
+cp -a "${DIFFROOT}"/* "${TMP_DIFFROOT}"
+
+"${CI_ROOT}/update-codegen.sh"
+echo "diffing ${DIFFROOT} against freshly generated codegen"
+ret=0
+diff -Naupr "${DIFFROOT}" "${TMP_DIFFROOT}" || ret=$?
+cp -a "${TMP_DIFFROOT}"/* "${DIFFROOT}"
+if [[ $ret -eq 0 ]]
+then
+  echo "${DIFFROOT} up to date."
+else
+  echo "${DIFFROOT} is out of date. Please run 'make code-gen'"
+  exit 1
+fi
+
+`))
```

Comment:
why do we need to generate this file with solo kit? 

i also don't see a `code-gen` make target anywhere.  where is this `verify_codegen.sh` coming from?